code/modules/clothing/spacesuits/rig.dm

```diff
@@ -335,7 +335,10 @@
 	flags = HEADCOVERSEYES | BLOCKHAIR | HEADCOVERSMOUTH | STOPSPRESSUREDMAGE | THICKMATERIAL
 	action_button_name = "Toggle Helmet Mode"
 	species_restricted = null
-	sprite_sheets = null
+	species_fit = list("Vox")

```

Comment:
Vox can already wear a blood-red hardsuit with no modification, so all this does is keep that functionality while applying a Vox-specific sprite once the helmet's worn.